ssz/constants.py

```diff
@@ -1 +1,5 @@
-SSZ_CHUNK_SIZE = 128
+CHUNK_SIZE = 32
+EMPTY_CHUNK = b"\x00" * CHUNK_SIZE
+
+SIZE_PREFIX_SIZE = 4
```

Comment:
`BYTES_PER_LENGTH_PREFIX`?

Reply:
See above comment. Admittedly `SIZE_PREFIX_SIZE` sounds funny, but at least it should be clear what it is supposed to mean.